# Get the map as a epthmap_humanoid_msgs/GreyScaleMap16bit
---
GreyScaleMap16bit map
